distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Java_Exports.enso

```diff
@@ -39,6 +40,10 @@ make_time_of_day_builder initial_size = TimeOfDayBuilder.new initial_size
 make_date_time_builder : Integer -> DateTimeBuilder
 make_date_time_builder initial_size = DateTimeBuilder.new initial_size
 
+## PRIVATE
+make_date_builder : Integer -> DateBuilder
+make_date_builder initial_size = DateBuilder.new initial_size
```

Comment:
I think it may be worth to move your `DateBuilderAdapter` into here. The issue of Date conversion at the polyglot boundary will happen with every instance of the builder, I think.